course/frontend/arrays/scripts/main.js

```diff
@@ -93,4 +93,39 @@ function recursiveArrayClone(source) {
     return result;
 }
 
-console.log('1) recursiveArrayClone([1, 2, 3, [1, 2, 3]])', res = recursiveArrayClone(arr), 'and should be a new array', res != arr, 'and sub-array should be a new array', arr[3] != res[3]);
\ No newline at end of file
+console.log('1) recursiveArrayClone([1, 2, 3, [1, 2, 3]])', res = recursiveArrayClone(arr), 'and should be a new array', res != arr, 'and sub-array should be a new array', arr[3] != res[3]);
+
+/*mostFrequentItem
+Write a JavaScript program to find the most
+ frequent item of an array.
+ */
+
+Array.prototype.mostFrenquentItem = function (array) {
```

Comment:
WELL done, @JaimeRC! just add a simple console.log test below to see how it works ,)